src/index.html

```diff
@@ -22,6 +22,465 @@
+
+      <nav class="header__nav">
+        <ul class="header__menu">
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link header__menu-link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__menu-item">
+            <a
+              href="#"
+              class="header__menu-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="iMac on white background"
+            class="card__image"
+          />
+          <div class="card__title">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+          </div>
+
+          <div class="card__code">
+            <p>Product code: 195434</p>
+          </div>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__review">Reviews: 5</p>
+          </div>
+
+          <div class="card__container">
+            <div class="card__container-price">
+              <p>Price:</p>
+            </div>
+            <div class="card__container-numbers">
+              <p>$2,199</p>
+            </div>
+          </div>
+
+          <button
```

Comment:
The 'data-qa' attribute with the value 'card-hover' is present on every 'BUY' button. This attribute is meant for testing, and if each button is identical, it could make it difficult to target specific buttons during testing. Ensure each 'data-qa' attribute has a unique value if needed for testing individual elements.